index.js

```diff
@@ -239,4 +239,17 @@ client.on("guildMemberAdd", member => {
 
 /* DEBUG */
 	
-});
\ No newline at end of file
+});
+
+function getPlacementString(place) {
+	// get last digit of the placement
+	let lastDigit = place.substr(place.length - 1, place.length);

```

Comment:
I tested this and you just need to convert `place` to a string and evaluate using `==` instead of `===` and it works perfectly: 

![image](https://user-images.githubusercontent.com/17626930/130382982-254aade8-e107-4ec6-b0e5-2ccc1d369f1b.png)

Reply:
Thanks for testing. Committed your suggestions and should be okay now. I'll set up a test environment soon for myself so you don't have to test for me.

Reply:
No worries! I also went back and realized that we might want to account for the small exception where numbers ending in 11, 12 or 13 should be appended with `th`. This should do it:

```
function getPlacementString(place) {
    let i = place % 10, j = place % 100;
    if (i == 1 && j != 11) { return `${place}st`; }
    if (i == 2 && j != 12) { return `${place}nd`; }
    if (i == 3 && j != 13) { return `${place}rd`; }
    return `${place}th`;
}
```